RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 - Renamed CHANGELOG.md to RELEASE-NOTES.txt
 - Open new note automatically upon creation
 - Open new note automatically upon creation [1566](https://github.com/Automattic/simplenote-electron/issues/1566)
+- Updated colors to use Color Studio in alignment with new design guidelines
```

Comment:
When I do a web search for `color studio` I don't find anything that would explain this. Could we add something that indicates what it is or more importantly, what impact it has on our customers?

Something like this maybe…
```
 - Updated colors to use Color Studio, the color palette for Automattic products
```